website/src/app/[lang]/[region]/survey/[recipient]/[survey]/questions.ts

```
@@ -24,15 +57,25 @@ export const welcomePage = (t: TranslateFunction, name: string) => {
 
 // Questions for onboarding survey (reused in other surveys)
 
+function getSimpleMapping(question: Question, t: TranslateFunction): Object {
+	return {
+		type: question.type,
+		name: question.name,
+		title: t(question.translationKey),
+		description: question.descriptionTranslationKey && t(question.descriptionTranslationKey),
+		choices:
+			question.choices && question.choices.length
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)